Automattic/studio
# Allow PHP version to be changed from Site Settings

Related https://github.com/Automattic/studio/issues/32

## Proposed Changes

Allows the PHP version to be changed from Site Settings:

https://github.com/Automattic/studio/assets/36432/dac23d45-ba44-43cc-9d59-6cc3cd4ea405

Also restarts the server after the PHP version is changed.

## Testing Instructions

1. Create a new site.
2. New site should launch with a default version of PHP 8.0.
3. Navigate to the Studio settings for the site.
4. Edit the PHP version to a different PHP version.
5. Verify the server restarts and the site is running the new PHP version.

## Pre-merge Checklist
